akka-distributed-cluster-docs/src/main/paradox/guide/3-active-active.md

```diff
@@ -1,10 +1,102 @@
 # Part 3: Active-active
 
-For an Active-Active shopping cart, the same service will run in different data centers or cloud regions - each called a replica.
+Active-active means that the same shopping cart is in memory in multiple locations, replicas, and can accept updates
+in all of those locations, so that a cloud region outage does not block users from continuing to use the service.  
+
+@extref[Akka Replicated Event Sourcing](akka:typed/replicated-eventsourcing.html) stores persisted events in a local database,
+without any need for replication capabilities in the database itself, the events are then replicated using gRPC
+transport from @extref[Akka Projection gRPC](akka-projection:grpc-replicated-event-sourcing-transport.html).
+
+The shopping cart will be eventually consistent, meaning that an update to a 
 
 ## Turning the shopping cart into a Replicated Entity
 
-FIXME show off RES API
+The API for writing a Replicated Event Sourced entity is mostly the exact same as the Event Sourced Behavior we already
+implemented our entity with. 
+
+The events from other replicas will be replicated and are passed to the event handler of the entity
+just like events that was written by the shopping cart instance itself, however there is no global ordering of the events, 
+so events may be seen in a different order than the wall clock order they happened, and in a different order in each replica, 
+especially in circumstances where there are outages or connectivity problems between the replicas.
+
+### Updating the cart contents
+
+Because of this the data structure must always arrive at the same state even if events arrive in a different order. To handle 
+this in the shopping cart we replace the `ItemAdded` and `ItemRemoved` events with a single `ItemUpdated` event
```

Comment:
We might need to align the grpc/shopping-cart and the replicated/shopping-cart so that we don't have to explain changes that are not really what we want to illustrate in the guide. The original cart had `AddItem` and validation that it can't add again. That doesn't work with RES. We can simplify the original to better fit the RES.

Reply:
Yeah, I'll "backport" `ItemUpdated` and maybe drop that validation.